src/main/java/com/refrigerator/item/service/ItemService.java

```diff
@@ -0,0 +1,43 @@
+package com.refrigerator.item.service;
+
+import com.refrigerator.item.dto.ItemCreateDto;
+import com.refrigerator.item.entity.Item;
+import com.refrigerator.item.repository.ItemCategoryRepository;
+import com.refrigerator.item.repository.ItemRepository;
+import com.refrigerator.state.repository.StateRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ItemService {
+
+    private final ItemRepository itemRepository;
+    private final ItemCategoryRepository itemCategoryRepository;
+    private final StateRepository stateRepository;
+
+    public ItemService(ItemRepository itemRepository, ItemCategoryRepository itemCategoryRepository, StateRepository stateRepository) {
+        this.itemRepository = itemRepository;
+        this.itemCategoryRepository = itemCategoryRepository;
+        this.stateRepository = stateRepository;
+    }
+
+    public void createItem(ItemCreateDto itemCreateDto) {
+        var category = itemCategoryRepository.findById(itemCreateDto.getCategoryId())
+                .orElseThrow(() -> new IllegalArgumentException("Category not found"));
+        var state = stateRepository.findById(itemCreateDto.getStateId())
+                .orElseThrow(() -> new IllegalArgumentException("State not found"));
```

Comment:
특수한 경우가 아니면 타입추론말고 직접 타입을 사용하면 좋을것 같아요
```suggestion
        Category category = itemCategoryRepository.findById(itemCreateDto.getCategoryId())
                .orElseThrow(() -> new IllegalArgumentException("Category not found"));
        State state = stateRepository.findById(itemCreateDto.getStateId())
                .orElseThrow(() -> new IllegalArgumentException("State not found"));
```